modules/EnsEMBL/Web/Component/Gene/GeneSummary.pm

```diff
@@ -55,6 +55,53 @@ sub content {
     );
   }
 
+  my ($has_mane_select, $has_mane_plus_clinical) = 0;
+  my ($mane_select, $mane_select_translation, $mane_plus_clinical, $mane_plus_clinical_translation);
+
+  ## add MANE info
+  if ($ensembl_select) {
```

Comment:
Do we need this conditional ?

Reply:
Will we always have an ensembl_select transcript? I was thinking that if we don't have an ensembl_select transcript then that also means there won't be a MANE select.

Reply:
No we can't, thanks